src/PureCloud.Client/Contracts/ISCIMApi.cs

```diff
@@ -4,13 +4,197 @@ namespace PureCloud.Client.Contracts;
+
+    /// <summary>Get a group.</summary>
+    /// <param name="groupId">The ID of a group. Returned with GET /api/v2/scim/v2/groups.</param>
+    /// <param name="attributes">Indicates which attributes to include. Returns these attributes and the "id", "displayName", and "meta" attributes. Use "attributes" to avoid expensive secondary calls for the default attributes.</param>
+    /// <param name="excludedAttributes">Indicates which attributes to exclude. Returns the default attributes minus "excludedAttributes". Always returns "id", "displayName", and "meta" attributes.</param>
+    /// <param name="ifNoneMatch">The ETag of a resource in double quotes. Returned as header and meta.version with initial call to GET /api/v2/scim/v2/groups/{groupId}. Example: "42". If the ETag is the same as the version on the server, returns 304 and does not return the requested resource.</param>
+    /// <param name="cancellationToken">Cancellation token</param>
+    Task<ScimV2Group> GetScimV2GroupAsync(string groupId, IEnumerable<string> attributes = null, IEnumerable<string> excludedAttributes = null, string ifNoneMatch = null, CancellationToken cancellationToken = default);
+
+    /// <summary>Get a list of groups.</summary>
+    /// <param name="filter">Filters results. If nothing is specified, returns all groups. Examples of valid values: "displayName eq groupName", "members.value eq userId".</param>
+    /// <param name="startIndex">The 1-based index of the first query result.</param>
+    /// <param name="count">The requested number of items per page. A value of 0 returns "totalResults".</param>
+    /// <param name="attributes">Indicates which attributes to include. Returns these attributes and the "id", "displayName", and "meta" attributes. Use "attributes" to avoid expensive secondary calls for the default attributes.</param>
+    /// <param name="excludedAttributes">Indicates which attributes to exclude. Returns the default attributes minus "excludedAttributes". Always returns "id", "displayName", and "meta" attributes.</param>
+    /// <param name="cancellationToken">Cancellation token</param>
+    Task<ScimGroupListResponse> GetScimV2GroupsAsync(string filter, int? startIndex = null, int? count = null, IEnumerable<string> attributes = null, IEnumerable<string> excludedAttributes = null, CancellationToken cancellationToken = default);
+
+    /// <summary>Get a resource type.</summary>
+    /// <param name="resourceType">The type of resource. Returned with GET /api/v2/scim/v2/resourcetypes.</param>
+    /// <param name="cancellationToken">Cancellation token</param>
+    Task<ScimConfigResourceType> GetScimV2ResourcetypeAsync(string resourceType, CancellationToken cancellationToken = default);
+
+    /// <summary>Get a list of resource types.</summary>
```

Comment:
Span summary over 3 lines where the text on the 2nd line as Microsoft standards. Do that where there is method documentation